src/model.py

```diff
@@ -331,9 +333,83 @@ def to_misp(self) -> List[MISPObject]:
 
         # filter out objects without any attributes
         to_return = list(filter(lambda x: bool(x.attributes), to_return))
-
         return to_return
 
+    def to_maco(self) -> model.ExtractorModel:
+        output = model.ExtractorModel(family=self.family)
+
+        for key in self.rsa_keys:
+            obj = model.ExtractorModel.Encryption(
+                algorithm="rsa",
+                public_key=str((key.n, key.e)),
```

Comment:
What is the recommended way to serialize a RSA public key? As far as I can see, I can only pass `str`